getting-started.ipynb

```diff
@@ -29,28 +29,36 @@
   "cells": [
     {
       "cell_type": "markdown",
-      "source": [
-        "# Getting Started"
-      ],
       "metadata": {
-        "id": "HgljXEAJEcFq"
-      }
-    },
-    {
-      "cell_type": "markdown",
+        "id": "P9VjKOirEcFu"
+      },
       "source": [
+        "<!--\n",
+        "---\n",
```

Comment:
The frontmatter is still visible even after putting the comments. This is because of the newline in line 36. Suggested change -
```
<!-----
```

Reply:
On second thought, if we remove the title of the notebook and comment it out as frontmatter, a user who tries to just view the notebooks in the examples repository won't be able to figure out what the notebook is about since comments won't be visible. What do you think we should do?

Reply:
For that, I guess we need to remove `# h1 title` when converting. Hugo uses `title` from frontmatter.

Reply:
yes let's keep `# h1 title` in a notebook and remove it on converting